src/redux/reducers/__tests__/scansReducer.test.js

```diff
@@ -78,7 +80,8 @@ describe('ScansReducer', () => {
       types.PAUSE_SCAN,
       types.RESTART_SCAN,
       types.START_SCAN,
-      types.GET_SCANS
+      types.GET_SCANS,
+      types.DELETE_SCAN
```

Comment:
required: around line `100` there is a `fulfilled` types check similar to `pending` and `error` just need to fill it in with the same `types.DELETE_SCAN` and update the related test snapshot. 

At a minimum it highlights that `DELETE_SCAN` is consistent in implementation with the other REST responses consumed by Redux